app/src/main/java/com/github/se/polyfit/ui/screen/SettingsScreen.kt

```diff
@@ -0,0 +1,93 @@
+package com.github.se.polyfit.ui.screen
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.KeyboardArrowRight
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.github.se.polyfit.R
+import com.github.se.polyfit.model.settings.SettingOption
+
+@Composable
+fun SettingsScreen(settings: List<SettingOption>, modifier: Modifier = Modifier) {
+  val context = LocalContext.current
+
+  Column(
+      modifier = modifier.fillMaxSize().testTag("SettingsScreen"),
+      horizontalAlignment = Alignment.CenterHorizontally,
+      verticalArrangement = Arrangement.Top) {
+        Text(
+            text = context.getString(R.string.settings),
+            style = MaterialTheme.typography.headlineMedium,
+            modifier = Modifier.padding(48.dp).testTag("SettingsTitle"))
+
+        LazyColumn(modifier = Modifier.testTag("SettingsList")) {
```

Comment:
im not sure it's strictly neccesary (since we won't ever have an empty setting screen), but maybe you can put a default text if there's no settings passed

Reply:
I agree with Mesha, not sure about the use of a lazyColumn, just a Column should be good

Reply:
Thats fair! I think it should be the same either way, and if we get too many settings it would be good to have a lazy column? I will keep it as is for now, but if we have functionality issue we can address later